### ISSUE TITLE
hi, Feiteng, does your asr work support Asynchronous Stochastic Gradient Descent? Thanks!

### ISSUE THREAD

**Issue by chaohui0** [lingochamp/kaldi-ctc#11]
hi, Feiteng, does your asr work support Asynchronous Stochastic Gradient Descent? Thanks!

**Comment by lifeiteng** [comment]
checkout ["Parallel training of Deep Neural Networks with Natural Gradient and Parameter Averaging", Daniel Povey, Xiaohui Zhang and Sanjeev Khudanpur, ICLR Workshop 2015](https://arxiv.org/abs/1410.7455)

**Comment by lifeiteng** [comment]
Yes.
checkout ["Parallel training of Deep Neural Networks with Natural Gradient and Parameter Averaging", Daniel Povey, Xiaohui Zhang and Sanjeev Khudanpur, ICLR Workshop 2015](https://arxiv.org/abs/1410.7455)
